### PR TITLE
GH-1107 Add Google Combo chart

### PR DESCRIPTION
**Pull request by dtsnell4**
Adds Google's Combo Chart to the framework.

Fixes GH-1107

Signed-off-by: Dan Snell <Dan.Snell@lexisnexis.com>

### CODE REVIEW

**Comment by dtsnell4** [comment]
@GordonSmith @mlzummo @jbrundage Please review

**Comment by dtsnell4** [comment]
http://rawgit.com/dtsnell4/Visualization/GH-1107_Google_Combo_Chart/demos/dermatology.html?src/google/Combo

File: test/googleFactory.js [hunk 1]
```diff
@@ -27,20 +27,31 @@
                 });
             }
         },
-        Scatter: {
```
Comment (GordonSmith) [inline test/googleFactory.js]: We should keep the Scatter sample?

Comment (dtsnell4) [inline test/googleFactory.js]: Its still there, its below....  not sure why github diffs that way sometimes.

**Pull request merged**